bip-0142.mediawiki

```diff
@@ -20,7 +20,12 @@ To define standard payment addresses for native segwit transactions to promote e
 === P2PKH segwit address ===
 
 The first new Bitcoin address format defined is specific to pay-to-public-key-hash segwit transaction. The scriptPubKey is a single push of version-0 witness program in this form,
-  <0x0076A914{20-byte-hash-value}88AC>
+  0x001976A914{20-byte-hash-value}88AC
+
```

Comment:
"The scriptPubKey is a single push of version-0 witness program in this form"

Why does it say the scriptPubKey is a single push? It's two pushes?